opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -129,17 +130,30 @@ def version(self):
     def schema_url(self):
         return self._schema_url
 
-    # FIXME check that the instrument name has not been used already
+    def _check_instrument_id(self, name, type_, unit, description):
+
+        instrument_id = "".join(
+            [name.strip().lower(), str(type_), unit, description]
+        )
```

Comment:
I think this will be more helpful id for user in the following warning message.
```suggestion
        instrument_id = ",".join(
            [name.strip().lower(), type_.__name__, unit, description]
        )
```